core/src/com/unciv/models/ruleset/unit/Promotion.kt

```diff
@@ -8,17 +8,16 @@ import com.unciv.models.translations.tr
 class Promotion : INamed{
     override lateinit var name: String
     var prerequisites = listOf<String>()
-    var effect = ""
```

Comment:
Don't deprecate this. Deprecation should be preceded by a deprecation warning.

Reply:
@yairm210 👌Now I don't edit this.
By the way, about **.joinToStringLanguageDependant()**, this fun should be written in the class Translation.